edge-functions/crypto/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Using Crypto in Edge Functions
+name: Using Crypto in Edge Functions and Middleware
```

Comment:
This feels redundant because Middleware is an edge function.

Reply:
It's not redundant. There's a difference between our Edge Middleware and our Experimental Edge functions (shown and distinguished in the docs)